profiles/Abdulwahid7027.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/110891341/211197326-db82ec3c-ad43-4e98-ab85-6332152f9dd8.png",
```

Comment:
I recommend more of an image like this. What do you think?

![image](https://user-images.githubusercontent.com/51391473/211386495-2f54c6c3-dc32-4f15-ac29-cc541f8b6d89.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/211386495-2f54c6c3-dc32-4f15-ac29-cc541f8b6d89.png",
```

Reply:
It's good.